testkit/src/api.rs

```diff
@@ -131,6 +133,7 @@ where
     access: ApiAccess,
     prefix: String,
     query: Option<&'b Q>,
+    modifier: Option<ReqwestModifier<'b>>,
```

Comment:
Modifier is not so clear name, maybe `decorator` or `middleware` is more good decision.